pensieve/experimenter.py

```diff
@@ -71,6 +72,12 @@ def started_since(self, since: dt.datetime) -> "ExperimentCollection":
         return cls([ex for ex in self.experiments if ex.start_date and ex.start_date >= since])
 
     def end_on_or_after(self, after: dt.datetime) -> "ExperimentCollection":
-        """All experiments that end on or after the specified date."""
+        """All live experiments that end on or after the specified date."""
         cls = type(self)
-        return cls([ex for ex in self.experiments if ex.end_date and ex.end_date >= after])
+        return cls(
+            [
+                ex
+                for ex in self.experiments
+                if ex.end_date and ex.end_date >= after and ex.status in ("Live", "Complete")
```

Comment:
We could also have a separate method for filtering out live experiments that gets applied to the result `end_on_or_after`. But this is also fine.